tests/from_msa/test_prg_builder.py

```diff
@@ -0,0 +1,146 @@
+from pathlib import Path
+from unittest import TestCase
+
+from Bio.Seq import Seq
+from Bio.SeqRecord import SeqRecord
+
+from make_prg.from_msa.prg_builder import PrgBuilder
+from tests.from_msa import make_alignment, MSA
+
+this_dir = Path(__file__).resolve().parent
+data_dir = this_dir.parent / "data" / "make_prg_from_msa"
+
+
+class TestConsensusString(TestCase):
+    def test_all_match(self):
+        alignment = make_alignment(["AATTA", "AATTA"])
+        result = PrgBuilder.get_consensus(alignment)
+        self.assertEqual(result, "AATTA")
+
+    def test_mixed_match_nonmatch(self):
+        alignment = make_alignment(["AAGTA", "CATTA"])
+        result = PrgBuilder.get_consensus(alignment)
+        self.assertEqual(result, "*A*TA")
+
+    def test_indel_nonmatch(self):
+        alignment = make_alignment(["AAAA", "A--A"])
+        result = PrgBuilder.get_consensus(alignment)
+        self.assertEqual(result, "A**A")
+
+    def test_IUPACAmbiguous_nonmatch(self):
+        alignment = make_alignment(["RYA", "RTA"])
+        result = PrgBuilder.get_consensus(alignment)
+        self.assertEqual(result, "**A")
```

Comment:
So if the bases are ambiguous, even if they match, we still consider then `*`?

Reply:
Yes because it doesn't tell you exactly which base to use as a non-variant base in the PRG